canvas/text.go

```diff
@@ -21,6 +21,11 @@ type Text struct {
 	Text      string         // The string content of this Text
 	TextSize  float32        // Size of the text - if the Canvas scale is 1.0 this will be equivalent to point size
 	TextStyle fyne.TextStyle // The style of the text content
+
+	// FontSource defines a resource that can be used instead of the theme for looking up the font.
+	//
+	// Since: 2.5
+	FontSource fyne.Resource
```

Comment:
What is the behavior if the FontSource doesn't actually match the TextStyle? eg FontSource is a bold font but TextStyle is default? I assume we will fall back to the theme but do we want to document this?

Reply:
Updated thanks

Reply:
I want to test this myself before I resolve this conversation, but looks good assuming that is the behavior in-practice

Reply:
Confirmed that the documented behavior (FontSource takes precedent over TextStyle if the source does not have glyphs for the style) matches the runtime behavior.